source/message/OP_MSG.rst

```diff
@@ -54,10 +54,10 @@ Usage
 MongoDB drivers SHOULD perform the MongoDB handshake using ``OP_MSG`` if an API
 version was declared on the client, but MAY decide to use ``OP_QUERY``.
 
-If no API version was declared, drivers that support MongoDB 3.4 and earlier
-MUST perform the handshake using ``OP_QUERY`` to determine if the node supports
-``OP_MSG``. Drivers that only support MongoDB 3.6 and newer MAY default to using
-``OP_MSG``.
+If no API version was declared, drivers that have historically supported MongoDB
+3.4 and earlier MUST perform the handshake using ``OP_QUERY`` to determine if the
```

Comment:
Although *IF* the user of the driver has specified a specific protocol, we can send OP_QUERY, so MUST doesn't seem right. As I understand it, OP_QUERY is to (always) be used if the driver intends to support <= MongoDB 3.4 servers AND the user has not specified a specific protocol, which is slightly different. (But, maybe this text isn't the right place to capture that?)

Reply:
Ah, I see if "no API version *was declared*". Ok. +1. (This captures the case where the user has requested a specific version.) +1